external-adapter/src/config.ts

```diff
@@ -21,6 +23,7 @@ export function getDefaultConfig(prefix = ''): Config {
   const apiKey = util.getEnv(ENV_API_KEY, prefix)
   return {
     apiKey,
+    returnVerboseResponse: !!util.getEnv(ENV_API_KEY, prefix),
```

Comment:
Oops this should use `ENV_API_VERBOSE_RESPONSE`.

This is for whether the adapter responses should include all of the response data for adapters that are making multiple requests. We talked about cases where if you have 50 addresses that you are fetching balances it might be a lot of response data. 

It is currently more of a balance specific use case, likely we will have more multiple request adapters in the future. All adapters _could_ also use this.

I'm open to moving it though, thoughts?